src/main/java/com/car/app/resource/CarResource.java

```diff
@@ -0,0 +1,70 @@
+package com.car.app.resource;
+
+import java.util.List;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.car.app.model.Car;
+import com.car.app.service.CarService;
+
+@Path("cars")
+public class CarResource {
+	private static Logger log = LoggerFactory.getLogger(CarResource.class);
```

Comment:
Please add log message for entering-exiting to every method as mentioned in the trello card.